src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -564,24 +565,29 @@
   </data>
   <data name="ImageResizer_ENCODER_TIFF_CCITT3.Content" xml:space="preserve">
     <value>CCITT3</value>
+    <comment>do not loc</comment>
```

Comment:
I find difficult to understand why a string that should not be localized has been placed in the resw file.

Reply:
Pattern shifted to everything goes in the resx file.

Reply:
One spot for all strings vs two